consumer/push_consumer.go

```diff
@@ -119,6 +119,9 @@ func NewPushConsumer(opts ...Option) (*pushConsumer, error) {
 
 	p.interceptor = primitive.ChainInterceptors(p.option.Interceptors...)
 
+	retryTopic := internal.GetRetryTopic(p.consumerGroup)
+	sub := buildSubscriptionData(retryTopic, MessageSelector{TAG, _SubAll})
+	p.subscriptionDataTable.Store(retryTopic, sub)
```

Comment:
Broadcast mode does not need to subscribe to retry topic

Reply:
fixed